google/src/main/scala/org/broadinstitute/dsde/workbench/google/GoogleIamDAO.scala

```diff
@@ -138,6 +138,19 @@ trait GoogleIamDAO {
                      memberType: MemberType,
                      rolesToRemove: Set[String]): Future[Boolean]
 
+  /**
+   *
+   * @param serviceAccountProject the google projecting which we're adding the IAM roles
```

Comment:
It'd be good to make it really clear that we're not adding roles to the Google project.

```suggestion
   * @param serviceAccountProject the google project where `serviceAccount` lives
```